### PR TITLE
Added scrollIntoView

### PR DESCRIPTION
**Pull request by inoshikag**
Added scrollIntoView to avoid the error 'element is not in the user's viewport'

### CODE REVIEW

File: lib/page_steps.js [hunk 1]
```diff
@@ -359,13 +361,17 @@ step("Scroll to <element>", async element => {
...
+  await taiko.evaluate(ele, (item) => item.scrollIntoView());
+  await taiko.click(ele);
 });
 
 step(["Click <element> <elementText>", "Click on <element> <elementText>", "Click <element> for <elementText>"],
   async (element, elementText) => {
   element = element + " with text " + helpers.getValue(elementText);
-  await taiko.click(await getPageElement(taiko, element));
+  const ele = await getPageElement(taiko, element);
+  await taiko.evaluate(ele, (item) => item.scrollIntoView());
```
Comment (erla-softrams) [inline lib/page_steps.js:373]: scroll into view has to be achieved with separate step `* Scroll to "element"`

File: lib/page_steps.js [hunk 2]
```diff
@@ -359,13 +361,17 @@ step("Scroll to <element>", async element => {
 });
 
 step(["Click <element>", "Click on <element>"], async element => {
-  await taiko.click(await getPageElement(taiko, element));
+  const ele = await getPageElement(taiko, element);
+  await taiko.evaluate(ele, (item) => item.scrollIntoView());
```
Comment (erla-softrams) [inline lib/page_steps.js:365]: scroll into view has to be achieved with separate step `* Scroll to "element"`

File: lib/page_steps.js [hunk 3]
```diff
@@ -47,18 +47,20 @@ step("Check text <text> exists", async text => {
...
-    await taiko.click(
-      await selectors.getElementBySelector(
-        eleType,
-        await selectors.getProximitySelector(proximity, text)
-      )
+    const ele = await selectors.getElementBySelector(
+      eleType,
+      await selectors.getProximitySelector(proximity, text)
     );
+    await taiko.evaluate(ele, (item) => item.scrollIntoView());
```
Comment (erla-softrams) [inline lib/page_steps.js:62]: scroll into view has to be achieved with separate step `* Scroll to "element"`

File: lib/page_steps.js [hunk 4]
```diff
@@ -47,18 +47,20 @@ step("Check text <text> exists", async text => {
 });
 
 step("Click <eleType> with text <text>", async (eleType, text) => {
-  await taiko.click(await selectors.getElementBySelector(eleType, text));
+  const ele = await selectors.getElementBySelector(eleType, text);
+  await taiko.evaluate(ele, (item) => item.scrollIntoView());
```
Comment (erla-softrams) [inline lib/page_steps.js:51]: scroll into view has to be achieved with separate step `* Scroll to "element"`

**Review by erla-softrams — CHANGES_REQUESTED** [review]

File: lib/page_steps.js (hunk 4 above)
Comment (inoshikag) [reply lib/page_steps.js:51]: We noticed this issue only after upgrading to the latest taiko version. It was working fine with the prev verison. If we use *Scroll to "element", there are so many places to be modified. So we had a chat with Murali and he said it is a good approach to tackle it,  particularly relevant when the element is not in view (due to size within view port, and needed scroll)

**Pull request merged**